assets/prototype/domain/eventEditor/containers/queries/priceTypes.js

```diff
@@ -4,13 +4,13 @@ export const PRICE_TYPE_ATTRIBUTES = gql`
 	fragment priceTypeAttributes on PriceType {
 		id
 		dbId
+		name
 		baseType
 		isBasePrice
 		isDeleted
 		isDiscount
 		isPercent
 		isTax
-		name
```

Comment:
bah you stinker! I was trying to put them in alphabetical order (except for the IDs) 😆 
but that's ok... not a big deal

Reply:
it shouldn't make any difference, 
I also think that we MUST keep them in alphabetical order

Reply:
I usually don't have strong opinion in shallow situations like this, but IMO keeping things in alphabetical order is critical to code hygiene

Reply:
Yes, putting them in alphabetical order is better 🤦‍♂